<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProtocolBufferElementVisualization</class>
 <widget class="QWidget" name="ProtocolBufferElementVisualization">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>PROTOCOL ELEMENT DETAILS:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="mainItemsLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="bufferIndexLabel">
     <property name="text">
      <string>BUFFER INDEX: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="wordTypeLabel">
     <property name="text">
      <string>WORD TYPE: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>430</width>
        <height>244</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="detailsLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>BYTES:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
